package.json

```diff
@@ -42,10 +46,6 @@
   "author": "Jordan Humphreys <jordan@zurb.com>",
   "license": "MIT",
   "dependencies": {
-    "exorcist": "^1.0.1",
-    "gulp-connect": "^5.7.0",
-    "gulp-rename": "^1.4.0",
-    "gulp-sourcemaps": "^2.6.5",
-    "gulp-uglify": "^3.0.2"
+    "exorcist": "^1.0.1"
```

Comment:
Even `exorcist` package seems to be also used only by gulp

Reply:
thanks, I have already moved